compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/jsx-outlining-child-stored-in-id.expect.md

```diff
@@ -50,26 +50,35 @@ export const FIXTURE_ENTRYPOINT = {
 ```javascript
 import { c as _c } from "react/compiler-runtime"; // @enableJsxOutlining
 function Component(arr) {
```

Comment:
nit: can we rewrite this fixture to destructure props i.e. `function Component({ arr })`?